### DISCUSSION TITLE
Download sleap using anaconda prompt

### DISCUSSION THREAD

**Discussion by CherieSun052** [talmolab/sleap#1426] [Help!]
I am trying to download sleap using anaconda prompt. I already downloaded mambaforge and conda. I am not sure if I am doing it right. What is the next step?

**Comment by roomrys** [comment]
Hi @CherieSun052,

[SLEAP v1.3.1+](https://github.com/talmolab/sleap/releases/tag/v1.3.1) requires [mamba](https://mamba.readthedocs.io/en/latest/installation.html#fresh-install) for faster package resolution. It is important not to install both anaconda and mamba __separately__ (it is ok - although not recommended - to [install mamba into an existing conda](https://mamba.readthedocs.io/en/latest/installation.html#existing-conda-install-not-recommended)).

So, after you have mamba installed, the [next steps](https://sleap.ai/installation.html#installation-methods) differ based on your OS...

#### Windows/Linux
Either open the mambaforge prompt if on Windows, or open a terminal if on Linux and run
```
mamba create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.1
```

#### Mac
Open a terminal and run
```
mamba create -y -n sleap -c conda-forge -c anaconda -c sleap sleap=1.3.1
```

Let us know where you are stuck and if you have any messages in your terminal.

Thanks,
Liezl
